client/src/main/webjar/program-form-dialog/program-form-dialog.module.js

```diff
@@ -6,7 +6,8 @@
         [
             "ngMaterial",
             "healthBam.templates",
-            "healthBam.requestedProgram"
+            "healthBam.program",
+            "healthBam.organization"
```

Comment:
Since this form will need these, I went ahead and made the module drag in both. It also allows them to be available in the app so they can be played with in the JS console.